src/erlcloud_ddb_impl.erl

```diff
@@ -114,12 +114,8 @@ timeout(1, #aws_config{timeout = undefined}) ->
     %% Shorter timeout on first request. This is to avoid long (5s) failover when first DDB
     %% endpoint doesn't respond
     1000;
-timeout(_, #aws_config{timeout = undefined}) ->
-    %% Longer timeout on subsequent requsets - results in less timeouts when system is
-    %% under heavy load
-    ?DEFAULT_TIMEOUT;
-timeout(_, #aws_config{timeout = Timeout}) ->
-    Timeout.
+timeout(_, #aws_config{} = Cfg) ->
```

Comment:
because of backward compatibility to original DDB impl. notice the difference for first retry attempt several lines above